forc-util/src/lib.rs

```diff
@@ -207,6 +207,20 @@ pub fn println_green(txt: &str) {
     println_std_out(txt, TermColor::Green);
 }
 
+pub fn println_light_blue(txt: &str) {
+    println_std_out(txt, TermColor::Rgb(40, 200, 255));
+}
+
+pub fn print_light_blue(txt: &str) {
+    let stdout = StandardStream::stdout(ColorChoice::Always);
+    print_with_color(txt, TermColor::Rgb(40, 200, 255), stdout);
+}
+
+pub fn print_light_green(txt: &str) {
+    let stdout = StandardStream::stdout(ColorChoice::Always);
+    print_with_color(txt, TermColor::Rgb(90, 255, 40), stdout);
+}
```

Comment:
It was recently discussed in #tooling on slack that we should switch from using light colours in the `forc init` / `forc new` output back to using default `tracing` log output as the output is difficult to read on some terminals and lighter themes.

Could you:

1. Remove these new functions.
2. Change the verbose output sections to use `tracing::debug!` instead.
3. Remove the `--verbose` output flags from `forc new` and `forc init` commands in favour of allowing users to specify the level of verbosity using the `RUST_LOG` environment variable (already provided by the `tracing` subscriber).

We can possibly revisit colouring output in a future dedicated PR, however when we do we should be more comprehensive in testing different terminals and colour themes (light, dark, etc) before moving ahead with it.

Reply:
This one is still unresolved @ra0x3

Reply:
@adlerjohn Doh! Forgot to push up the commit 🤦